### ISSUE TITLE
[BUG] opening cerain apps is buggy (UM41 flagship test ISO)

### ISSUE THREAD

**Issue by Atmois** [Ultramarine-Linux/ultramarine#58]
### Summary

when opening certain apps (images show calculator app) screen just goes funny and it sometime fixes itself but not always and fixes itself in many ways
![image](https://github.com/user-attachments/assets/f6c37e62-bdc0-46c8-915e-3bd3309e621d)
![image](https://github.com/user-attachments/assets/be1a7f5b-4a19-49b7-9b85-14e26d5499b7)
apps found to have the issue:
calculator
screen roller

### Steps to Reproduce

1. open cone of the listed apps
2. click around on screen and try drag until the image above happens (sometimes you can't drag it but its there)
3. press random keys and it sometimes fixes itself

### Release Version

41

### Edition / Desktop Environment / Hardware

Flagship 

### Checklist

- [X] I've checked for duplicates in this repository. (not required)
- [ ] I'm using the immutable/atomic version. (i.e. I think this issue might be specific to immutable.)
- [ ] This issue is not reproducible on Fedora Linux.
- [ ] This issue is not reproducible on other Linux distros.

**Comment by Atmois** [comment]
Oh also same happens with file roller

**Comment by madonuko** [comment]
Are you using a physical USB drive? This might be caused by lags / latency from data transfer

**Comment by Atmois** [comment]
> Are you using a physical USB drive? This might be caused by lags / latency from data transfer

Nope, Running it in a VM. I can try replicate it on an actual installation if that is needed.

**Comment by madonuko** [comment]
Not needed, I'll spin up a VM tomorrow morning.

**Comment by madonuko** [comment]
Can't reproduce, kinda suspect this happens from just your machine lagging
